forms-flow-api/src/formsflow_api/resources/draft.py

```diff
@@ -133,6 +133,18 @@ def put(draft_id: int):
 
             return response, status
 
+    @staticmethod
+    @auth.require
+    @profiletime
+    def delete(draft_id: int):
+        """Delete draft."""
+        try:
+            DraftService.delete_draft(draft_id)
+            return {"message": "Deleted"}, HTTPStatus.OK
```

Comment:
Draft deleted successfully  may be this is good for message

Reply:
draft success message updated